src/applications/personalization/dashboard/components/ManageYourVAHealthCare.jsx

```diff
@@ -41,12 +44,48 @@ const ScheduleAnAppointmentWidget = () => (
   </div>
 );
 
+const ScheduleAnAppointmentCernerWidget = () => (
+  <AlertBox
+    status="warning"
+    headline="Our records show that you're registered at Charlmers P. Wylie Ambulatory Care Center.  Starting April 10, 2020, providers at this center are using My VA Health to manage appointments."
+  >
+    <p>
+      You may need to sign in again to use VA health appointment tools. If you
+      do, sign in with the same account you used to sign in on VA.gov. You may
+      need to disable your browser's pop-up block so these tools can open.
+    </p>
+    <h4>Manage appointments with Chalmers P. Wylie provider</h4>
+    <a
```

Comment:
we tend to open external sites in a new window with `target="_blank"` and then add `rel="noopener noreferrer"`